doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
-     * Additionally, it is responsible for maintaining logical isolation between different instances of virtualization domain
-     * Some of the HW resources including networking resources shall be possible to be withheld from Virtualization domains to allow for scaling, spare parts and HW Composition within the HW Infrastructure Layer itself
-   * Virtualization layer shall provide the Cloud Tenants with an abstracted networking environment
-     * It is therefore responsible for maintaining isolation between Cloud Tenants
-
-An example of the layering described above is depicted in **Figure 1-3**, where the Virtualization layer manages the Overlay Networking (e.g., through VLAN allocation) and the HW Infrastructure manages the Underlay networking (e.g., through VxLAN VNI range allocations).
-<p align="center"><img src="./figures/RMCH-01_Network_Layering_v0-3.png" alt="Network Layering" title="Network Layering" width="100%"/></p>
-<p align="center"><b>Figure 1-3:</b> Network Layering</p>
+### 5.1.2 Networking Principles & Strategy
+
+This section introduces some networking related principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architecture 01 (OpenStack), Reference Architecture 02 (K8s) and down to Reference Implementations and Reference Conformance test suites under CNTT.
+
+•	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+> **Note:** Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
+
+•	**Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+-	Control, User and Management traffic types
+-	Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
+
+•	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+
+•	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+
+•	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
+
+•	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+•	**Workload diversity**: Should be capable to handle any VNF , CNF and BareMetal workloads.
```

Comment:
Maybe need to clarify:
1. Networking doesn't handle any workloads. Should we talk about traffic or connectivity of different types of workloads.
2. Not sure what you mean with BareMetal workloads? Workloads that run without any virtualization? Not sure we cover that in other parts of CNTT.
3. Instead of mentioning bare metal workloads, maybe we should mention Containers running in VMs vs. Containers running on bare metal?

Reply:
@ulikleber 
WRT 1: If we replace "...be _capable to handle_ any VNF..." with "be _**compatible with**_ any VNF...", would that address your concern?

WRT 2: Correct. I believe we do support bare metal. I know we've talked about it.

WRT 3: I wouldn't recommend specifying any particular embodiment of a workload. Using the generic term allows us to write in a workload embodiment agnostic manner.

Reply:
@ulikleber, @markshostak: I have changed to '•	**Workload diversity**: Should be capable to p=to provide connectivity to any type of workloads, including VNF , CNF and BareMetal workloads."